scripts/deploy.py

```diff
@@ -268,19 +313,49 @@ def _execute_deployment():
 
         print 'Returning to oppia/ root directory.'
 
+    release_version_library_url = (
```

Comment:
Actually you can move this up too, near the '.' check.

Reply:
Done.